package.json

```diff
@@ -32,6 +32,7 @@
   },
   "workspaces": [
     "packages/*",
+    "packages/analyzer/fixtures/11-nested-node-modules/01-basic",
```

Comment:
This is to generate a nested node_modules folder

Reply:
Can we reuse this test fixture instead? https://github.com/open-wc/custom-elements-manifest/blob/master/packages/analyzer/fixtures/02-inheritance/05-external-in-monorepo/monorepo/package.json

Reply:
Great, I removed the new test case and used the existing one. Annoyingly, the test case (even after I added a node_modules directory) passes on both the old and the new regex. My previous test failed on the old regex, but passed with the new one.

I can investigate further, but again it would probably complicate the repo structure a bit, what do you think?

Reply:
Yeah id prefer not to use the workspaces field for it, it muddies the water a bit